sshoot/main.py

```diff
@@ -188,6 +188,18 @@ def get_parser(self) -> ArgumentParser:
             type=str.split,
             help=_("extra arguments to pass to sshuttle command line"),
         )
+        create_parser.add_argument(
+            "--global-extra-opts",
+            action="store_true",
+            help=_("enable global extra arguments set in config.yaml"),
+        )
+        create_parser.add_argument(
+            "--no-global-extra-opts",
+            dest="global_extra_opts",
+            action="store_false",
+            help=_("disable global extra arguments set in config.yaml"),
+        )
+        create_parser.set_defaults(global_extra_opts=True)
```

Comment:
I think adding  this override flag per-config makes it harder to see the actual configuration.
I'd like it better to add a `--no-global-extra-opts` flag to the `start` command, to prevent them from being added when starting the profile.

Reply:
Hi, I moved the `--no-global-extra-opts` flag to the start, restart and get_command subcommands. I tested every command and it seems OK